src/java/mars/utils/Constans.java

```diff
@@ -0,0 +1,16 @@
+package mars.utils;
+
+public class Constans {
+    public static final double costU1 = 100;
+    public static final double weightU1 = 10000;
+    public static final double cargoU1 = 18000;
+
+    public static final double costU2 = 100;
+    public static final double weightU2 = 10000;
+    public static final double cargoU2 = 18000;
```

Comment:
costU2, weightU2, and cargoU2 have the same value than the U1 constants, the values for U2 should be

```
Rocket cost = $120 Million
Rocket weight = 18 Tonnes
Max weight (with cargo) = 29 Tonnes
```